yb-voyager/cmd/root.go

```diff
@@ -51,6 +58,12 @@ var rootCmd = &cobra.Command{
 			os.Exit(0)
 		}
 	},
+
+	PersistentPostRun: func(cmd *cobra.Command, args []string) {
+		if exportDir != "" && utils.FileOrFolderExists(exportDir) && cmd.Use != "version" && cmd.Use != "status" {
```

Comment:
Nit: Maybe you can just check if `lock != nil`.

Reply:
This is giving me mismatched types error. That's why sticking to the current code.